### ISSUE TITLE
Consulta-TP Strdup

### ISSUE THREAD

**Issue by juanhumphreys** [PI-ITBA/2024_02#220]
Buenas tardes, teniamos la consulta de si seria buen estilo utilizar el  #define _POSIX_C_SOURCE 200809L para tener el estandar POSIX y poder usar la funcion strdup o es mejor que escribamos nosotros la definicion de la misma en nuestro .c y no usemos el define.

**Comment by marcelogarberoglio** [comment]
Sólo código y funciones de la biblioteca estándar ANSI, no POSIX

**Comment by juanhumphreys** [comment]
perfecto muchas gracias.

**Comment by JuaniRaggio** [comment]
> Sólo código y funciones de la biblioteca estándar ANSI, no POSIX

pthreads creo es POSIX pero usando -std=c99 no nos dieron warnings, se puede usar?

**Comment by marcelogarberoglio** [comment]
No. El código equivalente a esa función lo han hecho (o deberían haberlo hecho) varias veces durante la cursada.

**Comment by marcelogarberoglio** [comment]
Creo que era más rápido escribir el código que este mensaje

**Comment by JuaniRaggio** [comment]
no hablo de strdup, hice la pregunta porque usamos pthread_create y pthread_join para optimizar que son de pthreads.h y esa biblioteca creo que es POSIX

**Comment by marcelogarberoglio** [comment]
No necesitan usar threads para este TP, no debería haber tareas concurrentes o que deben ejecutarse en paralelo.

**Comment by JuaniRaggio** [comment]
Nuestra implementacion tiene una query concurrente y funciona. Se puede usar o la sacamos?

**Comment by marcelogarberoglio** [comment]
No, no pueden usar POSIX.
